src/postgraphql/__tests__/pathgraphqlIntegrationSchemaExport-test.js

```diff
@@ -0,0 +1,28 @@
+jest.mock('fs')
+
+import withPgClient from '../../postgres/__tests__/fixtures/withPgClient'
+import createPostGraphQLSchema from '../schema/createPostGraphQLSchema'
+import exportPostGraphQLSchema from '../schema/exportPostGraphQLSchema'
+import { writeFileSync } from 'fs'
+
+test('exports a schema as JSON', withPgClient(async pgClient => {
+  const gqlSchema = await createPostGraphQLSchema(pgClient, ['a', 'b', 'c'])
+  await exportPostGraphQLSchema(gqlSchema, { exportPathJSON: '/schema.json' })
```

Comment:
You’ll need to extend the jasmine timeout to get this to work, see:

https://github.com/calebmer/postgraphql/blob/2f25ce8a9567e956f6dac55c237a1bf2e9e38003/src/postgraphql/__tests__/postgraphqlIntegrationSchema-test.js#L8-L9

Unfortunately, in CI, on our kitchen sink schemas `createPostGraphQLSchema` can be slow 😣